mvp-spec/operator-api.md

```diff
@@ -213,93 +213,34 @@ cat response-operatorO.json body-id-and-preferences.json | npx json --merge -e '
   "timestamp": 1639059692793,
   "signature": "message_signature_xyz1234",
   "body": {
-    "preferences": {},
-    "identifiers": []
-  }
-}
-```
-
-#### Signature
-
-Signature of the concatenation of:
-
-```
-sender + '\u2063' +
-receiver + '\u2063' +
-timestamp
-```
-
-### GET /v1/json/readOrGetNewId
-
-#### Request
-
-<!-- The query string below is generated with taking the request-publisherP.json file, removing body, and encoding it as query string:
-npx encode-query-string -nd `cat request-publisherP.json | npx json -e 'this.body = undefined' -o json-0`
--->
-
-```http
-GET /v1/json/readOrGetNewId?sender=publisherP.com&timestamp=1639057962145&signature=message_signature_xyz1234
-```
-
-#### Signature
-
-Signature of the concatenation of:
-
-```
-sender + '\u2063' +
-receiver + '\u2063' +
-timestamp 
-```
-
-#### Response
-
-Note: list of identifiers **cannot** be empty.
-
-<!-- Update this code block with:
-cat response-operatorO.json body-id-and-preferences.json | npx json --merge -e 'this.body.preferences = {}'
--->
-
-```json
-{
-  "sender": "operatorO.com",
-  "timestamp": 1639059692793,
-  "signature": "message_signature_xyz1234",
-  "body": {
-    "preferences": {},
-    "identifiers": [
-      {
-        "version": 1,
-        "type": "prebid_id",
-        "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
-        "source": {
-          "domain": "operator0.com",
-          "timestamp": 1639643110,
-          "signature": "prebid_id_signature_xyz12345"
-        }
+    "newIdentifier": {
+      "version": 1,
+      "type": "prebid_id",
+      "value": "560cead0-eed5-4d3f-a308-b818b4827979",
+      "source": {
+        "domain": "operator0.com",
+        "timestamp": 1639643110,
+        "signature": "prebid_id_signature_xyz12345"
       }
-    ]
+    }
   }
 }
 ```
 
-#### Signature
+##### Response signature
 
 Signature of the concatenation of:
 
 ```
 sender + '\u2063' +
 receiver + '\u2063' +
-preferences.source.signature + '\u2063' +
-identifiers[0].source.signature + '\u2063' +
-identifiers[1].source.signature + '\u2063' +
-...
-identifiers[n].source.signature + '\u2063' +
+newIdentifier.source.signature + '\u2063' +
```

Comment:
If you accept the previous comment about using "identifiers", don't forget to update it here.